tsp/tsp_client.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, base_url):
         '''
         Constructor
         '''
-        self.base_url = base_url
+        self.base_url = base_url + '/' if not base_url.endswith('/') else base_url
```

Comment:
can this be inverted... I think it makes it easier to read:

self.base_url = base_url if base_url.endswith('/') else base_url + '/'

Reply:
You're right, done !